src/main/java/com/github/zetten/maven/xvfb/AbstractXvfbMojo.java

```diff
@@ -128,17 +136,44 @@
 	protected String fbdir;
 
 	/**
+	 * The number of seconds to wait for return value when stopping Xvfb process.
+	 */
+	@Parameter(defaultValue = "13", required = false)
+	protected long stopTimeoutInSeconds;
+
+	/**
+	 * Additional arguments to the Xvfb process.
+	 */
+	@Parameter(required = false)
+	protected List<String> xvfbArgs;
```

Comment:
I think Maven's default is to split on commas if the parameter is an array-type.

Allowing both an arg array and the raw arg line is good enough for now though - maybe it'll get cleaned up sometime in the future if anyone complains :smile: